tools/linker-process/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+load("//tools:defaults.bzl", "ts_library")
+
+package(default_visibility = ["//visibility:public"])
+
+ts_library(
+    name = "sources",
+    srcs = glob(["**/*.ts"]),
+    deps = [
+        "@npm//@angular/compiler-cli",
+        "@npm//@babel/core",
+        "@npm//@babel/traverse",
+        "@npm//@types/node",
+    ],
+)
+
+nodejs_binary(
```

Comment:
Could you add a comment explaining the purpose of this target?

Reply:
👍